.github/workflows/publish-python.yaml

```diff
@@ -98,45 +104,44 @@ jobs:
           - '3.9'
           - '3.10'
     steps:
-      - name: checkout
+      - name: "Checkout"
         uses: actions/checkout@v2
         with:
           submodules: true
       - uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python }}
-      - run: python --version
-      - run: pip install --user -r requirements.txt
-      - run: bash generate.sh
-        shell: bash
-      - run: pip install --user wheel
-      - name: build wheel
-        run: python setup.py -v bdist_wheel
+
+      - name: "Generate bdk.py"
+        run: |
+          python --version
```

Comment:
Just a nit.. but you don't need this line, you probably only had it in for debugging.